src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -1257,14 +1295,16 @@ bool AddRule(DiagnosticDescriptor rule, string? category)
                             effectiveSeverity = DiagnosticSeverity.Warning;
                         }
 
-                        if (shippedReleaseData != null)
+                        if (releaseTrackingDataAndVersion != null)
                         {
                             isEnabledByDefault = isEnabledRuleForNonDefaultAnalysisMode;
-                            var maxVersion = shippedReleaseData.Value.version;
+                            var maxVersion = releaseTrackingDataAndVersion.Value.isShippedVersion ?
+                                releaseTrackingDataAndVersion.Value.version :
+                                ReleaseTrackingHelper.UnshippedVersion;
```

Comment:
Cosmetic preference but feel free to keep it as-is:

```suggestion
                            var maxVersion = releaseTrackingDataAndVersion.Value.isShippedVersion
                                ? releaseTrackingDataAndVersion.Value.version
                                : ReleaseTrackingHelper.UnshippedVersion;
```

Reply:
Yeah, it seems the code style for multiline ternary expressions has many divided preferences. We should really have a code style rule that can be enforced on build to ensure consistency. @CyrusNajmabadi mentioned he plans to author an IDE code style analyzer for it. I'll wait for such an enforcement before changing code styles.